package-lock.json

```diff
@@ -1,9204 +1,8 @@
 {
   "name": "pulsar-client",
   "version": "1.6.0-rc.0",
-  "lockfileVersion": 2,
```

Comment:
Seems we removed a lot of code, what were these codes used for?

Reply:
Thanks, revert this change